receiver/sqlserverreceiver/logs-documentation.md

```diff
@@ -0,0 +1,16 @@
+| sqlserver.total_physical_reads  | Total number of physical reads performed by executions of this plan since it was compiled.<br><br>Always 0 when querying a memory-optimized table.                                                                                                                                                                 | int    |   |   |   |   |   |   |   |
+| sqlserver.execution_count       | Number of times that the plan has been executed since it was last compiled.                                                                                                                                                                                                                                        | int    |   |   |   |   |   |   |   |
+| sqlserver.total_grant_kb        | The total amount of reserved memory grant in KB this plan received since it was compiled. Always 0 for querying a memory-optimized table.<br><br>Applies to: SQL Server 2016 (13.x) and later versions.                                                                                                            | int    |   |   |   |   |   |   |   |
+| sqlserver.query_hash            | Hex encoded hash value calculated on the query and used to identify queries with similar logic. You can use the query hash to determine the aggregate resource usage for queries that differ only by literal values.                                                                                               | string |   |   |   |   |   |   |   |
+| sqlserver.query_plan_hash       | Hex encoded hash value calculated on the query execution plan and used to identify similar query execution plans. You can use query plan hash to find the cumulative cost of queries with similar execution plans.<br><br>Always 0x000 when a natively compiled stored procedure queries a memory-optimized table. | string |   |   |   |   |   |   |   |
+| db.query.text            | Obfuscated text of the SQL query.<br><br>Is NULL for encrypted objects. This one is defined in Opentelemetry Semantic Convention, so we are using the naming following the convention. https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/db.md                                                                                                                                                                                                                                           | string |   |   |   |   |   |   |   |
+| sqlserver.query_plan            | Obfuscated query plan                                                                                                                                                                                                                                                                                              | string |   |   |   |   |   |   |   |
```

Comment:
There are several markdown columns are totally empty, like `|   |   |   |   |   |   |`. Could we remove them?

Reply:
updated